modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -1,39 +1,126 @@
+* `persistent` to store logs to disk  in `/var/log/journal/`. systemd creates the directory if it does not exist. 
+* `auto` to store logs in  in `/var/log/journal/` if the directory exists. If it does not exist, systemd temporarily stores logs in `/run/systemd/journal`.
+* `none` to not store logs. systemd drops all logs.
+<6> Specify the timeout before synchronizing journal files to disk for *ERR*, *WARNING*, *NOTICE*, *INFO*, and *DEBUG* logs. 
+systemd immediately syncs after receiving a *CRIT*, *ALERT*, or *EMERG* log. Default is `1s`.
+<7> Specify the maximum size the journal can use. Defaults to `8g`.
+<8> Specify how much disk space systemd must leave free. Defaults to `20%`.
+<9> Specify the maximum size for individual journal files stored persistently in `/var/log/journal`. Defaults to `10M`.
++
+[NOTE]
+====
+If you removing the rate limit, you might see increased CPU utilization on the
+system logging daemons as it processes any messages that would have previously
+been throttled.
+====
++
+For more information on systemd settings, see link:https://www.freedesktop.org/software/systemd/man/journald.conf.html[https://www.freedesktop.org/software/systemd/man/journald.conf.html]. The default settings listed on that page might not apply to {product-title}.
++
+// Defaults from https://github.com/openshift/openshift-ansible/pull/3753/files#diff-40b7a7231e77d95ca6009dc9bcc0f470R33-R34
+
+. Convert the `journal.conf` file to base64:
++
+----
+$ export jrnl_cnf=$( cat /journald.conf | base64 -w0 )
+----
 
-. Add the following to the *_/etc/systemd/journald.conf_* file:
+. Create a new MachineConfig for master and/or worker and add the `journal.conf` parameters:
++
+For example: 
 +
+[source,yaml]
 ----
-# Disable rate limiting
-RateLimitInterval=1s
-RateLimitBurst=10000
-Storage=volatile
-Compress=no
-MaxRetentionSec=30s
+
+....
+
+config:
+  storage:
+    files:
+    - contents:
+        source: data:text/plain;charset=utf-8;base64,${jrnl_cnf}
+        verification: {}
+      filesystem: root
+      mode: 0644 <1>
+      path: /etc/systemd/journald.conf <2>
+  systemd: {}
 ----
+<1> Set the permissions for the `journal.conf` file. We recommend setting to `0644` permissions. 
+<2> Specify the path to the base64-encoded `journal.conf` file.
 
-. Restart the services:
+. Create the MachineConfig:
 +
 ----
-$ systemctl restart systemd-journald.service
-$ systemctl restart rsyslog.service
+$ oc apply -f <filename>.yaml
 ----
 +
-These settings account for the bursty nature of uploading in bulk.
+The controller detects the new MachineConfig and generates a new `rendered-worker-<hash>` version. 
 
-After removing the rate limit, you might see increased CPU utilization on the
-system logging daemons as it processes any messages that would have previously
-been throttled.
+. Use `oc describe machineconfigpool/worker` to monitor the status of the rollout of the new rendered configuration to each node:
```

Comment:
Not critical, but suggest removing the command from the sentence (you don't do that in the steps above). Just

```
Monitor the status of the rollout of the new rendered configuration to each node:
```